src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/RequestContextPassTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Danil Pyatnitsev
+ * Date: 03.02.2020
+ * Time: 22:12
+ */
```

Comment:
Header should be removed

Reply:
thanks, I just all headers, and now fabbot.io is green